hips/hip-9999.md

```diff
@@ -0,0 +1,90 @@
+---
+hip: "9999"
+title: "Add export-values directive (similar to import-values)"
+authors: [ "Jan von Loewenstein <jan.von.loewenstein@sap.com>", "Johannes Dillmann <j.dillmann@sap.com>", "Pavel Busko <pavel.busko@sap.com>", "Philipp Stehle <philipp.stehle@sap.com>", "Ralf Pannemans <ralf.pannemans@sap.com>", "Sumit Kulhadia <sumit.kulhadia@sap.com>" ]
+created: "2022-02-01"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+
+Composing Helm charts is currently limited because values can only be distributed from a subchart to the composite chart via the [`import-values`][import-values] directive.
```

Comment:
```suggestion
Composing Helm charts using dependencies is currently not forming a closed abstraction. One cannot define a chart with a defined set of values and distribute those values to subcharts. Users of a parent chart instead have to provide values to the subcharts directly. Alternatively, users can provide `global` values, but in that case the subcharts need to be aware of the fact that they are used as a dependency. In addition, `global` values are available globally as the name suggests and hence multiple subcharts have to have aligned value keys. As a result, the parent chart and it's dependencies are rather tightly coupled.

With the `import-values` directive, there is a mechanism in Helm to use the default values of a subchart as a value in the parent chart and define the name under which the subchart value is available in the parent chart.
```

Reply:
@c0d1ngm0nk3y The change is now marked as outdated, but I guess my sugestion still applies. WDYT?